gradle.properties

```diff
@@ -4,11 +4,11 @@ main_version=1
 major_version=7
 minor_version=0
 minecraft_version=1.16.5
-forge_version=36.2.4
+forge_version=36.2.26
 mcp_mappings=1.16.5
 mcp_channel=official
 vampirism_mcversion=1.16.5
-vampirism_version=1.8.3
+vampirism_version=1.8.9
```

Comment:
Since the TeamLapen/Vampirism@3fa8dc89 is not published yet, i used a local version